server/models/item.ts

```diff
@@ -0,0 +1,86 @@
+import { Model, Transaction } from 'objection';
+
+export interface IItemCreateInput {
+  id: string;
+  name: string;
+  pokemonId: string;
+  price: number;
+  happiness: number;
+  imageUrl: string;
+}
+
+export interface IItemEditInput {
+  name?: string;
+  pokemonId?: string;
+  price?: number;
+  happiness?: number;
+  imageUrl?: string;
+}
+
+export default class Item extends Model {
+  static pickJsonSchemaProperties = true;
+  static tableName = 'item';
+
+  static jsonSchema = {
```

Comment:
```suggestion
  static jsonSchema: JsonSchema = {
```
to make sure that you're following the right structure for the schema definition